cedar-policy/src/api.rs

```diff
@@ -735,6 +734,40 @@ impl Authorizer {
     }
 }
 
+/// Errors that can occur during authorization
+#[derive(Debug, Diagnostic, PartialEq, Eq, Error, Clone)]
+pub enum AuthorizationError {
+    /// An error occurred when evaluating a policy.
+    #[error("while evaluating policy `{id}`: {error}")]
+    PolicyEvaluationError {
+        /// Id of the policy with an error
+        id: ast::PolicyID,
```

Comment:
We should `#[doc(hidden)]` this to avoid confusion with the actual getter function. That should be fair game for a minor release since you can still access it if really want to.

Reply:
Good call. Done in the last commit.